overlord/configstate/configcore/corecfg.go

```diff
@@ -73,6 +73,7 @@ func Run(tr Conf) error {
 	if err := validateExperimentalSettings(tr); err != nil {
 		return err
 	}
+	// FIXME: ensure the user cannot set "core seed.loaded"
```

Comment:
Perhaps testing the change by just trying to tr.Get and making sure it hasn't changed before/after?

Reply:
I need to look into this, we only get the "new" configuration from tr.Get() in configcore.Run() so to compare we need to get the state from tr.State(), get a new transaction with the current configuration and check if it is missing in the current config but is set in the new config and error only in this case.